packages/lodestar/src/network/util.ts

```diff
@@ -100,3 +101,12 @@ export function clearMultiaddrUDP(enr: ENR): void {
   enr.delete("ip6");
   enr.delete("udp6");
 }
+
+export function prettyPrintPeerId(peerId: PeerId): string {
+  const id = peerId.toB58String();
+  return `${id.substr(0, 2)}...${id.substr(id.length - 6, id.length)}`;
+}
+
+export function getAgentVersionFromPeerStore(peerId: PeerId, peerStore: PeerStore): string {
+  return peerStore.get(peerId)?.metadata?.get("AgentVersion")?.toString() || "";
```

Comment:
as discussed on the planning meeting last Friday, I changed this function to use the metadatabook instead of accessing via `peerstore.get()`.  when i benchmark the function in it's latest form (via the test added in this PR) all 2000 of the added peers take less than 1ms to access the `AgentVersion` except for roughly about 9 of said peers on average.

cc @wemeetagain